bundles/framework/mydata/Flyout.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AccountTab } from './view/Account/AccountTab';
+import { Tabs } from 'antd';
+import { MyDataHandler } from './handler/MyDataHandler';
+import { MyViewsTab } from './view/MyViews//MyViewsTab';
+import { FlyoutContent } from './FlyoutContent';
+import { PublishedMapsHandler } from './handler/PublishedMapsHandler';
+import { MyViewsHandler } from './handler/MyViewsHandler';
+import { PublishedMapsTab } from './view/PublishedMaps/PublishedMapsTab';
+
+/**
+ * @class Oskari.mapframework.bundle.personaldata.Flyout
+ */
+Oskari.clazz.define('Oskari.mapframework.bundle.personaldata.Flyout',
+
+    /**
+     * @method create called automatically on construction
+     * @static
+     * @param {Oskari.mapframework.bundle.personaldata.PersonalDataBundleInstance} instance
+     *     reference to component that created the flyout
+     */
+
+    function (instance) {
+        this.instance = instance;
+        this.container = null;
+        this.state = null;
+
+        this.template = null;
+        this.templateTabHeader = null;
+        this.templateTabContent = null;
+        this.uiHandler = new MyDataHandler(() => this.update());
+        this.publishedMapsHander = new PublishedMapsHandler(() => this.update(), this.instance);
+        this.myViewsHandler = new MyViewsHandler(() => this.update());
```

Comment:
Remove the listener param from these as they are added through `addTab()` that already calls `handler.addStateListener(() => this.update());`

Reply:
Actually we don't really need or want references to these handlers here so we could generate them in `startPlugin()` on the `addTab()` calls?